danesfield/core/tests/test_telesculptor.py

```diff
@@ -0,0 +1,52 @@
+import subprocess
+
+from django.test import override_settings
+import pytest
+from rdoasis.algorithms.models import Algorithm, AlgorithmTask, Dataset
+from rgd.models import ChecksumFile
+from rgd_3d.models import Tiles3D
+from rgd_fmv.models import FMV
+
+from danesfield.core.tasks import run_telesculptor
+
+
+@pytest.mark.django_db
+@override_settings(
+    BROKER_BACKEND='memory',
+    CELERY_ALWAYS_EAGER=True,
+    CELERY_EAGER_PROPAGATES_EXCEPTIONS=True,
+)
+def test_telesculptor_pipeline(dataset_factory, fmv: FMV):
+    try:
+        subprocess.run('nvidia-smi', stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+    except FileNotFoundError:
+        pytest.skip('NVIDIA GPU not detected.')
```

Comment:
After writing this test I realized it won't run in CI, since GitHub Actions runners don't have GPUs. I don't see any harm in keeping it around so it can be run locally, though.